pkg/chrootarchive/diff_unix.go

```diff
@@ -98,7 +108,7 @@ func applyLayerHandler(dest string, layer archive.Reader, options *archive.TarOp
 		return 0, fmt.Errorf("ApplyLayer json encode: %v", err)
 	}
 
-	cmd := reexec.Command("storage-applyLayer", dest)
+	cmd := reexec.Command("docker-applyLayer", dest)
```

Comment:
This has to agree with `pkg/chrootarchive/archive_unix.go`.

Reply:
fixed